config/initializers/feature_toggle.rb

```diff
@@ -4,4 +4,8 @@ def self.supported_housing_schemes_enabled?
 
     false
   end
+
+  def self.startdate_two_week_validation_enabled?
+    true
```

Comment:
This is enabled for all environments right?

Reply:
(Comment was saying production only)

Reply:
Yep, for now this is enabled everywhere. The purpose of the feature flag is to enable it to eventually be turned off in staging when testing new forms changes for next year.

Reply:
@smeny I forgot to update the PR description yesterday when I updated the feature flag